datajunction-ui/webpack.config.js

```diff
@@ -100,7 +102,7 @@ module.exports = {
       template: path.resolve(__dirname, 'public', 'index.html'),
     }),
     new webpack.DefinePlugin({
-      REACT_APP_DJ_URL: 'http://localhost:8000',
+      "process.env": JSON.stringify(process.env)
     }),
     // new MiniCssExtractPlugin({
```

Comment:
nit: why so many commented out lines in this file?

Reply:
Updated this file to remove the commented lines. @shangyian I'll follow your lead on the UI testing suite. I started to get familiar with the component testing setup that you have there so maybe I'll try adding those for these new pages 👀